### PR TITLE
ran npm run build-list & add take.md

### PR DESCRIPTION
**Pull request by kingdavidmartins**
similar to lodash compact method

Use `.slice()` to create a slice of the array with n elements taken from the beginning.

```js
const take = (arr, n) => n === undefined ? arr.slice(0, 1) : arr.slice(0, n);

// take([1, 2, 3], 5) -> [1, 2, 3]
// take([1, 2, 3], 0) -> []
```

https://lodash.com/docs/4.17.4#take

### CODE REVIEW

File: snippets/take.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Take
+
+Use `.slice()` to create a slice of the array with n elements taken from the beginning.
+
+```js
+const take = (arr, n) => n === undefined ? arr.slice(0, 1) : arr.slice(0, n);
```
Comment (Chalarangelo) [inline snippets/take.md]: Remove the condition `n === undefined`, use a default parameter for `n` in the definition (`(arr, n = 1)`). Apart from that, this should be fine.

Comment (kingdavidmartins) [reply snippets/take.md]: Copy, sometimes forget I can do that also. Thanks

**Comment by kingdavidmartins** [comment]
updated

File: README.md [hunk 2]
```diff
@@ -743,6 +744,17 @@ const tail = arr => arr.length > 1 ? arr.slice(1) : arr;
 // tail([1]) -> [1]
 ```
 
+### Take
+
+Use `Array.slice()` to create a slice of the array with `n` elements taken from the beginning.
```
Comment (Chalarangelo) [inline README.md]: As a note for the future refer to the first bullet under *Additional guidelines and conventions regarding snippets* of [CONTRIBUTING.md](https://github.com/Chalarangelo/30-seconds-of-code/blob/master/CONTRIBUTING.md#additional-guidelines-and-conventions-regarding-snippets) when referring to `Array` methods. Apart from that, solid work! Thanks for contributing yet again!

**Pull request merged**